backend/app/api/endpoints/knowledge.py

```
@@ -382,6 +407,11 @@ def list_documents(
             user=current_user,
             knowledge_base_id=knowledge_base_id,
         )
+    except ConsumerAccessDeniedError as e:
+        raise HTTPException(
+            status_code=status.HTTP_403_FORBIDDEN,
+            detail=str(e),
+        )
```

Ruff (0.15.2)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)